services/blockchain-indexer/methods/dataService/dposVotesReceived.js

```diff
@@ -15,18 +15,17 @@
  */
 const {
 	getVoters,
```

Comment:
Can we please rename the methods to `getVotesSent` and `getVotesReceived` everywhere to remove any confusion?